tools/refract-filter.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# This is just a
```

Comment:
Is this take from somewhere? Can we have the full file (with comments) and URL, so that we can update it later easily if it's updated upstream?